src/rust/rust-proto-new/src/graplinc/common/v1beta1.rs

```diff
@@ -0,0 +1,197 @@
+pub use std::time::{
+    Duration,
+    SystemTime,
+};
+use std::time::{
+    SystemTimeError,
+    UNIX_EPOCH,
+};
+
+use bytes::{
+    Buf,
+    BufMut,
+};
+use prost::Message;
+pub use uuid::Uuid;
+
+use crate::{
+    protobufs::graplinc::common::v1beta1::{
+        Duration as _Duration,
+        Timestamp as _Timestamp,
+        Uuid as _Uuid,
```

Comment:
I think the consensus is that rather than using underscore for the protos we should just add Proto at the end. This came up in the rust-proto PR.

Reply:
The reason I used an underscore here was to signify that these are _internal_ to this crate, and it's the same as what we do in `python-proto`. In python a leading underscore has a special meaning whereas in rust we can (and do here) use a more powerful mechanism to restrict visibility. We can change it to say `Proto` at the end, but does it actually make anything clearer? I kind of like the underscore here because it makes these things _ugly_ (which they are) and clearly marks them as "not for public consumption".

Reply:
I don't personally care either way - I felt similar when I used underscore. Just saying that the convention we've landed on is to add Proto.

Reply:
renamed everything e.g. `_Foo` --> `FooProto`